docs/config.rst

```diff
@@ -776,6 +776,11 @@ uppercase.
                                     note that with the default Mongo layer,
                                     setting this to ``False`` will result in an
                                     error. Defaults to ``True``.
+``NORMALIZE_DOCUMENT_FOR_PATCH``    If ``True``, the patch document will be
+                                    normalized according to schema. This means
+                                    if a field is not included in the patch
+                                    body, it will be reset to the default value
+                                    in its schema.
```

Comment:
Two things here:
1) I would add an explanation of what happens when set to False (a field with a default schema value will be ginored if absent aka, won't be reset);
2)  I would add "Defaults to True at the end of the paragraph.